Palo Alto Networks/paloalto-ngfw/ingest/parser.yml

```diff
@@ -715,16 +717,14 @@ stages:
           user_agent.os.version: "{{parsed_event.message.ClientOSVersion}}"
           user.name: "{{parsed_event.message.User or parsed_event.message.suser or parsed_event.message.PanOSSourceUserName or parsed_description.message.user}}"
           paloalto: >-
-            { 
```

Comment:
This should've worked without changes, but jinja stops parsing stuff in field whenever it mets `range` or `namespace` functions. I'm sure there are others, but these two are reason this parser doesn't work.